### PR TITLE
Milestone5 event handlers

### PR DESCRIPTION
**Pull request by DanielMitiku**
Full functionalities of the calculator app have been implemented.

### CODE REVIEW

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @DanielMitiku  :clap: 

### Awesome job so far!! :+1: 

##### You have done wonderfully well, you passed all checklist for this project :+1: 

1. **Kindly watch and listening carefully to the video and audio code review below;**
[Video Link](https://www.loom.com/share/78b6e13547db4158b1bf8448fc6c8589)
**_Hope it helps clarify the issues we are having ......_**

2. README file not detailed enough. (you can add steps to git clone your repo, necessary installation and live link)
Remove the default `readme` content
Kindly watch the video tutorial on how to create live link below 👇;
[Video Link](https://www.loom.com/share/76088842a2d14322bbcf65b884f4d42d)
checkout out the below link for help, free to follow the template from the pathwright
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/

**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: src/logic/calculate.js [hunk 1]
```diff
@@ -24,34 +39,41 @@ const calculate = ({ total, next, operation }, bName) => {
       } if (bName === '.') {
```
Comment (bellom) [inline src/logic/calculate.js:39]: whenever `bName === .`  we should be able to return `0.` not `.`
I think the app doesn't look professional like that right?

Screenshot from your app below 👇
image

File: src/logic/calculate.js [hunk 2]
```diff
@@ -2,16 +2,31 @@ import operate from './operate';
...
+            next: '',
+            operation: bName,
+          };
+        }
         return {
-          total: next,
+          total: (next === '') ? total : next,
           next: '',
           operation: bName,
         };
```
Comment (bellom) [inline src/logic/calculate.js:32]: Whenever the `AC` button is  clicked, we should have `0` as the result right.... shown in the loom video

**Comment by DanielMitiku** [comment]
I've made the changes for the comments given. Thank you.

**Review by shloch — APPROVED** [review]
## Hello @DanielMitiku 

PROJECT COMPLETE
## 👏👏👏👏👏👏👏👏👏👏👏👏

SEE YOU ON NEXT PROJECT

**Comment by DanielMitiku** [comment]
Thank you!

**Pull request merged**